core/spec/models/spree/variant_spec.rb

```diff
@@ -3,6 +3,13 @@
 require 'rails_helper'
 
 RSpec.describe Spree::Variant, type: :model do
+  context "new instance" do
```

Comment:
@rubenochiavone I think this example readability may benefit from using Rspec [implicit subject](https://relishapp.com/rspec/rspec-core/v/3-5/docs/subject/implicitly-defined-subject). In Rspec the default `subject` is an instance of the class being specified (similar to `described_class.new(args)` but without any arguments... see [here](https://relishapp.com/rspec/rspec-core/docs/metadata/described-class)):

```it { expect(subject).to be_invalid }```

or, even more [concisely](https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax):

```it { is_expected.to be_invalid }```

Reply:
Whoa, that's great and works perfectly! Do you mind if I add a reference to implicit subject?

```ruby
it { is_expected.to be_invalid } # implicitly defined subject, see https://relishapp.com/rspec/rspec-core/v/3-5/docs/subject/implicitly-defined-subject
```

Reply:
I'm 👍 on using this syntax but 👎 on adding a code comment. I think you could add a reference in the commit description instead (please keep it all in a single commit 🙂).

Reply:
Ok, cool. :+1:

Reply:
Done